app/assets/javascripts/admin/addon/components/modal/uploaded-image-list.js

```diff
@@ -0,0 +1,27 @@
+import Component from "@glimmer/component";
+import { action } from "@ember/object";
+import { tracked } from "@glimmer/tracking";
+
+export default class UploadedImageList extends Component {
+  @tracked
+  images = this.args.model.value?.length
+    ? this.args.model.value.split("|")
+    : [];
+
+  @action
+  remove(url, event) {
+    event?.preventDefault();
```

Comment:
can `event` arg be null-ish?

Reply:
This was copied over from previous, safe to remove